uberpoet/blazeprojectgen.py

```diff
@@ -186,14 +168,19 @@ def gen_app_main(self, app_node, module_index):
         language = module_index[importing_module_name]["language"]
         class_key = first_key(file_index.classes)
         class_index = first_in_dict(file_index.classes)
-        function_key = class_index[0]
+        function_key = first_in_dict(class_index)[0]
```

Comment:
in FileResult class functions are now stored by their function type hence this change.
Before:
```
{0: [1,2,3]}
```
After:
```
{0: {'swift_only': [1,2,3]}}
```